_install-and-configure/configuration/search-settings.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Search settings
+parent: Configuring OpenSearch
+nav_order: 80
+---
+
+# Search settings
+
+OpenSearch supports the following search settings:
+
+- `search.max_buckets` (Dynamic, integer): The maximum number of aggregation buckets allowed in a single response. Default is `65535`. 
+
+- `search.phase_took_enabled` (Dynamic, Boolean): Enables returning phase-level `took` time values in search responses. Default is `false`. 
+
+- `search.allow_expensive_queries` (Dynamic, Boolean): Allows or disallows expensive queries. For more information, see [Expensive queries]({{site.url}}{{site.baseurl}}/query-dsl/index/#expensive-queries).
+
+- `search.cancel_after_time_interval` (Dynamic, Time unit): A cluster-level setting that specifies the maximum amount of time that a search request can run before it is canceled at the shard level. If the `timeout` is specified in the search request, it takes precedence over this setting. Default is `-1`.
```

Comment:
I looked into the code. I'm not convinced that the per-request `timeout` parameter interacts with this one, but I might have missed it.

As far as I know, this one is arguably the "timeout nuclear option", where we just "hang up" on a running request after the given value and try to cancel any associated tasks.

Reply:
Thanks for a thorough review, @msfroh! I implemented your comments.